lib/backbeat/Metrics.js

```diff
@@ -71,10 +77,36 @@ class Metrics {
             } else {
                 queryString = `${site}:${op}`;
             }
+            if (hasGlobalKey) {
+                return this._redisClient.get(queryString, (err, res) => {
+                    if (err) {
+                        return done({
+                            message: `Redis error: ${err.message}`,
```

Comment:
This is an anti-pattern, why is this isn't an instance of `Error`?

Reply:
This is debt that will need to be refactored eventually. The error log if we just return the error returned by redis will not properly log from the BackbeatServer

Reply:
Indeed, I've followed the pattern of logging that is used in this file. We could eventually refactor those as well.